pubspec.yaml

```diff
@@ -14,7 +15,7 @@ dependencies:
   hooks_riverpod: ^2.4.10
   flutter_hooks: ^0.20.5
   nanoid: ^1.0.0
-  firebase_core: ^2.27.1
+  firebase_core: 2.27.1
```

Comment:
Just curious, why is locking the version of firebase_core necessary ?

Reply:
Because the new update contains a breaking change that might not be stabilized and could change in the future.
Apparently, they ran into some issues on their library and committed a code that was push to everyone as a minor patch, but wasn't one in reality.

In the meantime, we lock it and see what will happen to the library in the future.